src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_createSpecFileTemplate.yaml

```
@@ -0,0 +1,8 @@
+post:
+  deprecated: true
+  summary: Create a RPM Spec File Template for a specified package.
+  description: |
+    Create a RPM Spec File Template for a specified package. This endpoint
+    is broken and will be removed. See https://github.com/openSUSE/open-build-service/issues/9707
```

```suggestion
    is broken and will be removed. See [https://github.com/openSUSE/open-build-service/issues/9707](https://github.com/openSUSE/open-build-service/issues/9707).
```
Could we make this link clickable?